<?xml version="1.0" encoding="utf-8"?>
<launch>
	<arg name="fx" default="604.7211303710938" />
	<arg name="fy" default="603.2431640625" />
	<arg name="sub_depth" default="/camera/aligned_depth_to_color/image_raw" />
	<arg name="sub_boxes" default="/traffic_sign/darknet_ros/bounding_boxes" />
	<arg name="sub_image" default="/camera/color/image_raw" />
	<arg name="pub_image" default="/traffic_sign_located/image_raw" />
	<node name="traffic_sign_located" pkg="vision" type="traffic_sign_located" output="screen">
		<param name="fx" value="$(arg fx)" />
		<param name="fy" value="$(arg fy)" />
		<remap from="/camera/color/image_raw"  to="$(arg sub_image)" />
		<remap from="/traffic_sign_located/image_raw"  to="$(arg pub_image)" />
		<remap from="/camera/aligned_depth_to_color/image_raw"  to="$(arg sub_depth)" />
		<remap from="/traffic_sign/darknet_ros/bounding_boxes"  to="$(arg sub_boxes)" />
	</node>
</launch>

